src/components/ShowDefinitions.jsx

```diff
@@ -1,22 +1,22 @@
 import React from "react";
 const { Rune } = window;
 
-export default function ShowDefinitions({ definitionsObject }) {
+export default function ShowDefinitions({ definitionsObject, isJudge }) {
 	const { definitions, stateFlag } = definitionsObject;
 
 	function onclickHandler(e) {
 		Rune.actions.incrementScore(e);
 	}
 
 	const renderDefinition = () => {
-		if (stateFlag === true) {
-			const definitionsToBeShown = Object.keys(definitions);
+		const definitionsToBeShown = Object.keys(definitions);
+		if (isJudge && stateFlag === true) {
```

Comment:
If `stateFlag` is a boolean, we don't need to write `stateFlag === true`, we can simply look at `stateFlag`.

So `isJudge && stateFlag === true` can just be `isJudge && stateFlag.